src/Behat/Behat/Context/Reader/AnnotatedContextReader.php

```diff
@@ -132,6 +133,18 @@ private function readDocBlockCallees($class, ReflectionMethod $method, $docBlock
     }
 
     /**
+     * Merges multiline strings (strings ending with "\")
+     *
+     * @param string $docBlock
+     *
+     * @return string
+     */
+    private function mergeMultilines($docBlock)
+    {
+        return preg_replace("#\\\\$\s*\*\s*([^\\\\$]+)#m", '$1', $docBlock);
```

Comment:
you should replace ``\s*`` by ``\s*+`` to make the quantifiers possessive. This will avoid lots of useless backtracking.

and I think the ``[^\\\\$]+`` quantifier could also be possessive.